items/de.json

```
@@ -0,0 +1,202 @@
+{
+  "42": "42",
+  "air": "Luft",
+  "alchemistry": "Alchemie",
```

```suggestion
  "alchemistry": "Alchemistry",
```
Here it's the game itself 😄